itwm_example/mco/mco.py

```diff
@@ -133,21 +136,36 @@ class WeightedEvaluator(HasStrictTraits):
     """
     single_point_evaluator = Instance(ISinglePointEvaluator)
     weights = List(Float)
+    scale_factors = List(Float)
     parameters = List(BaseMCOParameter)
 
-    def __init__(self, single_point_evaluator, weights, parameters):
+    def __init__(self, single_point_evaluator, weights, scale_factors, parameters):
         super(WeightedEvaluator, self).__init__(
             single_point_evaluator=single_point_evaluator,
             weights=weights,
+            scale_factors=scale_factors,
             parameters=parameters,
         )
 
     def _score(self, point):
+
+        """
```

Comment:
I would just remove the old score function rather than keep it around as a comment

Reply:
Great, cheers - I've just committed a couple of changes to address the flake8 and redundant _score function